docs/usage/user-stories/swissquote.md

```diff
@@ -0,0 +1,235 @@
+
+Every company that’s been around for sometime has that old project still running.
+People talk about it laughingly but become livid when a request comes to change anything in it.
+
+![A person sweating profusely at the idea of having to touch an old project](../assets/images/swissquote_sweating_guy.jpg)
+
+There is a fix to make on that project, the last person working on it left three years ago.
+
+## Dependencies at Swissquote
+
+I’ve identified three main approaches people use to upgrade their dependencies at Swissquote:
+
+1. Critical fixes only; update dependencies that have a CVE attached
+1. Opportunistic; also known as the boy scout rule, leave the project with more up-to-date dependencies than you found it
+1. Update your dependencies regularly; manually or with a tool
+
+### Critical Fixes only
+
+Every once in a while, a library gets an update that fixes a vulnerability.
+Our security team is on the lookout for critical vulnerabilities and will quickly warn all impacted teams if an update needs to be done.
+
+We also configured [GitHub’s Dependabot Alerts](https://docs.github.com/en/code-security/dependabot/dependabot-alerts/about-dependabot-alerts) that will inform teams that a vulnerability has been discovered and which version they should upgrade to to be safe (more on this later).
+
+This approach, while it mostly works, can be very risky.
+If the project hasn’t been touched for quite a while, the number of dependencies to update can be high depending on the vulnerability and the time to update will increase significantly.
+
+### Opportunistic
+
+This rule is simple, when you receive a task on a project, the first step is to upgrade its dependencies.
+Once that is done, you can go ahead with the implementation of your task.
+
+This approach is similar to fixing a bit of tech debt with every business project you take on, it helps to stay ahead in case a vulnerability comes up and keep a baseline for all your projects.
+
+### Update your dependencies regularly
+
+This one sounds easy, every few weeks you check what’s outdated, bump the versions and run your CI.
+If it succeeds, you push the change.
+Maybe there is even a way to automate that?
+
+#### Let's give Renovate a try
+
+One day in November 2019, I discovered that [Renovate](https://www.mend.io/free-developer-tools/renovate/) provides a Docker image that you can run on premise with your own package registries.
+The app creates Pull Requests automatically to inform you of dependencies updates and the CI can then build it automatically so you know if it’s safe to merge or not.
+
+On our first try, we enabled 30 repositories, a cron task was running every hour to create Pull Requests.
+
+We received 700 Pull Requests in the first month, it was a never ending Pull Request whack-a-mole every time we merged one, another replaced it.
+
+![A cat playing Whack-a-mole with a finger](../assets/images/swissquote_cat_whack_a_mole.gif)
```

Comment:
Can we remove or replace this image? It will mean an extra 3MB download for anyone cloning the repo

Reply:
ah indeed that's way too big, I'll see what I can do, maybe just put a static image

Reply:
Replaced by a frame from the gif, and compressed all images with ImageOptim